test/benchgc/CMakeLists.txt

```diff
@@ -39,3 +39,4 @@ add_subdirectory("src/benchgc/mlir")
 add_subdirectory("src/benchgc/linalg")
 add_subdirectory("src/benchgc/tensor")
 add_subdirectory("src/benchgc/arith")
+add_subdirectory("src/benchgc/pattern")
```

Comment:
I would like to refactor this part with a loop, instead of appending a new one

Reply:
I think it‘s unnecessary to refactor, let's keep this simple